test/metabase/models/user_test.clj

```diff
@@ -194,12 +194,12 @@
                        (-> (invite-user-accept-and-check-inboxes! :google-auth? true)
                            (select-keys ["crowberto@metabase.com" "some_other_admin@metabase.com"])))))))))))
 
- (testing "if sso enabled and password login is disabled, email should send a link to sso login"
-   (mt/with-premium-features #{:disable-password-login}
-     (mt/with-temporary-setting-values [enable-password-login false]
-       (ldap.test/with-ldap-server
-         (invite-user-accept-and-check-inboxes! :invitor default-invitor , :accept-invite? false)
-         (is (seq (mt/regex-email-bodies #"/auth/login"))))))))
+  (testing "if sso enabled and password login is disabled, email should send a link to sso login"
+    (mt/with-premium-features #{:disable-password-login}
+      (mt/with-temporary-setting-values [enable-password-login false]
+        (ldap.test/with-ldap-server
+          (invite-user-accept-and-check-inboxes! :invitor default-invitor , :accept-invite? false)
+          (is (seq (mt/regex-email-bodies #"/auth/login"))))))))
```

Comment:
My 2c about changes like that — I am not sure which linters/prettifiers everyone uses for clojure, but probably different. Let's avoid changes like that because they unnecessarily change the history.

Reply:
Oopsy.
I didn't even notice that because I have the whitespace turned off in GH UI.

Thanks